packages/elements/src/components/ui/value-table/ValueTableItem.tsx

```diff
@@ -9,14 +10,24 @@ export interface ValueTableItemProps {
   label: string;
   labelIndent?: boolean | number;
   value: string | number | boolean | ReactNode | undefined;
-  alignValue?: BoxProps["justifyContent"];
+  justifyContentLabel?: BoxProps["justifyContent"];
+  justifyContentValue?: BoxProps["justifyContent"];
+  labelWidth?: Property.Width;
```

Comment:
```suggestion
  labelWidth?: ComponentPropsWithoutRef<"td">["width"];
```
This will remove the dependency on `csstype`, so you can remove it.

Reply:
fixed